app/api/sul_bib/authorship_api.rb

```diff
@@ -104,7 +104,8 @@ def get_publication_via_pubmed!(pmid)
       # @param [String] sw_id ScienceWire ID
       # @return [Publication]
       def get_publication_via_sciencewire!(sw_id)
-        Publication.find_or_create_by_sciencewire_id(sw_id) ||
+        Publication.find_by(sciencewire_id: sw_id) ||
+          SciencewireSourceRecord.get_pub_by_sciencewire_id(sw_id) ||
           log_and_error!("The ScienceWire:#{sw_id} publication was not found either locally or at ScienceWire.")
       end
```

Comment:
`Publication.find_by(sciencewire_id: sw_id)` 👍 

- wondering whether the `SciencewireSourceRecord.get_pub_by_sciencewire_id(sw_id)` needs to change in this PR to focus only on hitting the db without any SW-API calls or if that might happen in a follow on PR that will avoid hitting the SW-API.